news/2018-05-26-beatmap-spotlights-april-2018.md

```diff
@@ -0,0 +1,144 @@
+
+[![](/wiki/shared/news/2018-05-26-beatmap-spotlights-april-2018/der-wald.jpg)](http://osu.ppy.sh/beatmapsets/581787)
+
+An amazing set of top-quality maps by [Ascendance](https://osu.ppy.sh/users/2931883), [Spectator](https://osu.ppy.sh/users/702598) and [Hareimu](https://osu.ppy.sh/users/4138746)! Ascendance, as the mapper of the top difficulty, intentionally uses tricky yet decent anti-flows throughout the map, which heavily challenge reading skills and tapping precision. 
+
+The lower two difficulties are also neatly organized with delicate and friendly patterns, making the set enjoyable for fruit lovers of all levels. Don't forget to check it out!
+
+[![](/wiki/shared/news/2018-05-26-beatmap-spotlights-april-2018/akaito.jpg)](http://osu.ppy.sh/beatmapsets/610994)
+
+It's time we could enjoy 1/2 jump party! Surrounded by 1/4-rhythm-based overdoses today, [RoseusJaeger](https://osu.ppy.sh/users/67098409) and [Ascendance](https://osu.ppy.sh/users/2931883) are surprisingly gifting us a newly approved mapset, "Akaito" by Rib, with mostly 1/2 patterns. This map features convenient flow and proper emphasis throughout the whole difficulty. Meanwhile it is also able to come up with different patterns that make it colorful and catchy. With Rib's smooth voice, you are about to love playing this map.
+
+## osu!mania
+
+[![](/wiki/shared/news/2018-05-26-beatmap-spotlights-april-2018/lesson.jpg)](http://osu.ppy.sh/beatmapsets/704987)
+
+"Don't mess the timing with your eyes. Feel the rhythm with your soul!" [Lenfried-](https://osu.ppy.sh/users/5314573) brings us here the probably most enjoyable and fun mapset which consist only out of easy difficulties! The BASIC consist mostly of 2/1 and 1/1 pattern which got placed in order to the vocals from the music. It's perfect for every new player.
+
+The CHALLENGE difficulty is where the fun begins! The vocal tells you how to play and then the pattern repeat again but with a incredibly high slider velocity, high enough that the notes become unreadable. You have to remember which pattern you played before and listen to the vocal to hit them accurate!
+
+This mapset really deserve it to be the best of the month April. Well done Lenfried-! Would be cool to see more maps like this from you!
+
+[![](/wiki/shared/news/2018-05-26-beatmap-spotlights-april-2018/deublithick.jpg)](http://osu.ppy.sh/beatmapsets/725078)
+
+[\_underjoy](https://osu.ppy.sh/users/2235750) came back with his undoubtfully insane 7K mapset with Gekikara Mania's "Deublithick". The spread features with three extreme difficulties, including [Blocko](https://osu.ppy.sh/users/4075092) as a guest mapper and an Insane difficulty accompanied by [capsafangirl](https://osu.ppy.sh/users/4308015), which present extreme streaming burst at the 1/3 posterior section of the song, along with complex usage of multiple chord stream jump hands.
```

Comment:
features with three -> features three

Reply:
mapper and -> mapper, and
since it's a long sentence a comma probably works better

Reply:
can a word other than posterior be used so that people that were born after 1700 can understand it